integrationTests/testProcessorNode.go

```diff
@@ -839,6 +839,7 @@ func (tpn *TestProcessorNode) createFullSCQueryService() {
 			BlockGasLimit:                  tpn.EconomicsData.MaxGasLimitPerBlock(tpn.ShardCoordinator.SelfId()),
 			GasSchedule:                    gasSchedule,
 			ArgBlockChainHook:              argsHook,
+			InitialEpoch:                   0,
```

Comment:
no need to initialize everywhere as it is 0 by default.

Reply:
Replaced with `EpochNotifier`.